tests/ut/test_algorithm.py

```diff
@@ -314,3 +287,23 @@ def test_initialization_with_invalid_params(self, param, value, exc_type):
         await lock_manager.destroy()
 
         redis.clear_connections.assert_called_once_with()
+
+    @pytest.mark.asyncio
+    async def test_auto_extend(self):
```

Comment:
Is it possible to speed up this test?

Reply:
it's slow because I use real_sleep, set internal_lock_timeout shorter can make this faster. I'll fix this